source/user-manual/reference/ossec-conf/global.rst

```diff
@@ -554,6 +557,24 @@ This setting toggles whether to query the external Wazuh Cyber Threat Intelligen
 | **Allowed values** | yes, no |
 +--------------------+---------+
 
+.. _reference_forward_to:
+
+forward_to
+^^^^^^^^^^
+
+Specifies the name of the socket where the output will be redirected. The socket must be defined previously.
```

Comment:
Can we expand on "must be defined previously"? What do we want to say here?

Reply:
Yes, that's correct. I use the same definition as this https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/localfile.html#target